app/grandchallenge/challenges/models.py

```diff
@@ -664,6 +665,65 @@ def create_default_pages(self):
             permission_level=Page.ALL,
         )
 
+    def create_default_onboarding_tasks(self):
+        OnboardingTask.objects.create(
+            challenge=self,
+            title="Create Phases",
+            description="Define, name and create the different phases of the challenge.",
+            responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+            deadline=self.created + timedelta(weeks=1),
+        )
+        OnboardingTask.objects.create(
+            challenge=self,
+            title="Define Inputs and Outputs",
+            description="To support, communicate the required input and output data formats for participant's algorithms.",
+            responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+            deadline=self.created + timedelta(weeks=3),
+        )
+        OnboardingTask.objects.create(
+            challenge=self,
+            title="Create Archives",
+            description="Create an archive per algorithm-type phase for the challenge.",
+            responsible_party=OnboardingTask.ResponsiblePartyChoices.SUPPORT,
+            deadline=self.created + timedelta(weeks=3),
+        )
+        OnboardingTask.objects.create(
+            challenge=self,
+            title="Upload Data",
+            description="Prepare and upload necessary datasets to archives, contact support if no archives are available.",
+            responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+            deadline=self.created + timedelta(weeks=5),
+        )
+        OnboardingTask.objects.create(
+            challenge=self,
+            title="Example Algorithm",
+            description="Implement and document a baseline example algorithm for participants to use as a reference.",
```

Comment:
Do they know that support will provide them with a template for this and the evaluation containers?

Reply:
I hope that by the time this plays they have had a onboarding meeting.

Maybe a new 'task' for the orgs: receive challenge pack with examples and a task for support: create challenge pack?

tbh I think we'll be revisiting the task definitions a few times the next few months to get it right.

Reply:
I think it would be enough to add something along the lines of:
```suggestion
            description="Implement and document a baseline example algorithm for participants to use as a reference. Use the provided challenge pack as starting point for this.",
```

Reply:
Same should hold for the evaluation method descr.

Reply:
I would suggest adding the onboarding meeting as a task, that way everybody can see in the admin if that has happened already.